pkg/cmd/dcos_test.go

```diff
@@ -49,3 +52,27 @@ Use "dcos [command] --help" for more information about a command.
 
 	require.Equal(t, expectedHelp, out.String())
 }
+
+func TestPluginEnv(t *testing.T) {
+	cluster := config.NewCluster(nil)
+	cluster.SetURL("https://dcos.example.com")
+	cluster.SetACSToken("abc")
+	cluster.Config().Set("hello.world", "foo")
+	cluster.Config().Set("hallo.world", "foo")
+
+	env := pluginEnv("/path/to/me", "hello", logrus.DebugLevel, cluster)
+
+	require.Contains(t, env, "DCOS_CLI_EXECUTABLE_PATH=/path/to/me")
+	require.Contains(t, env, "DCOS_CLI_VERSION="+version.Version())
+	require.Contains(t, env, "DCOS_VERBOSITY=2")
+	require.Contains(t, env, "DCOS_URL=https://dcos.example.com")
+	require.Contains(t, env, "DCOS_ACS_TOKEN=abc")
+	require.Contains(t, env, "DCOS_HELLO_WORLD=foo")
+	require.NotContains(t, env, "DCOS_HALLO_WORLD=foo")
+}
+
+func TestCmdConfigEnvKey(t *testing.T) {
+	require.Equal(t, "DCOS_HELLO_WORLD", cmdConfigEnvKey("hello", "world"))
+	require.Equal(t, "DCOS_HELLO_WORLD_FOO", cmdConfigEnvKey("hello-world", "foo"))
+	require.Equal(t, "DCOS_HELLO_AROUND_THE_WORLD", cmdConfigEnvKey("hello", "around_the_world"))
```

Comment:
How about parametrized test, I'm not strong on this since this test looks simple enough but @drewvanstone loves parametrized tests :)

```go
func TestCmdConfigEnvKey(t *testing.T) {
	testCases := []struct {
		expected  string
		cmdName  string
		configKey string
	}{
		{"DCOS_HELLO_WORLD", "hello", "world"},
		{"DCOS_HELLO_WORLD_FOO", "hello-world", "foo"},
		{"DCOS_HELLO.WORLD_FOO", "hello.world", "foo"},
		{"DCOS_HELLO_AROUND_THE_WORLD", "hello", "around_the_world"},
	}
	for _, tc := range testCases {
		t.Run(tc.expected, func(t *testing.T) {
			assert.Equal(t, tc.expected, cmdConfigEnvKey(tc.cmdName, tc.configKey))
		})
	}
}
```

Reply:
We have test tables in some unit tests, I just thought it's overkill in this case.